Makefile

```diff
@@ -7,7 +7,8 @@ CGO_ENABLED:=0
 GOMODVENDOR:=
 
 CFLAGS:=-O2 -Wall
-LDFLAGS:=-static -s # strip C binaries
+LDFLAGS:= -s # strip C binaries
```

Comment:
Is the intent to also remove static linking for vsockexec?

Reply:
I think it would be good to have both linked dynamically for consistency, but I can add static linking back to vsockexec if you'd prefer that we make minimal changes to that binary.